cc-statistics/README.md

```diff
@@ -0,0 +1,228 @@
+# Common Crawl statistics notebooks
+
+### Introduction
+
+The goal of this repo is to allow users to easily and interactively view statistics about the Common Crawl data set.
+
+### Requirements
+
+Python environment (3.x)
+
+Jupyter Notebook
+
+### Quick Start
+Clone the [cc-webgraph](https://github.com/commoncrawl/cc-webgraph) repository and run the java tools with `mvn package`. 
```

Comment:
https://github.com/commoncrawl/cc-webgraph/blob/master/src/script/webgraph_ranking/install_webgraph.sh

Reply:
> do we need cc-webgraph

Although the functionality of this might be repeated in the Jython section, still think it might be valuable in case someone does not intend on running the pyWebGraph stuff? Not sure, let's discuss this later

Reply:
Maybe we could also get rid of all the scripts to run the Java tools, see https://github.com/commoncrawl/cc-webgraph/pull/2 - would also significantly simplify running Jython.